jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ReproducibleImageTest.java

```diff
@@ -111,7 +111,7 @@ public void testTarballStructure() throws IOException {
 
   @Test
   public void testManifest() throws IOException {
-    try (InputStream input = Files.newInputStream(imageTar.toPath())) {
+    try (InputStream ignored = Files.newInputStream(imageTar.toPath())) {
```

Comment:
Can this just be removed? Or can we at least explain why we're doing this?

Reply:
Good point